typings/index.d.ts

```diff
@@ -844,6 +844,11 @@ export interface Telegraf<C extends ContextMessageUpdate> extends Composer<C> {
    */
   telegram: Telegram
 
+  /**
+   * Use this property to extend context and support interface C
+   */
+  context: C
```

Comment:
Why `C`? Why not `Cat` or `Carpet`?

This definitely should ba called `TelegrafContext` just like it called in the sources. Anyway not `C`!

Reply:
Because `Telegraf<C extends ContextMessageUpdate>`

Reply:
There is also something `Composer`

Reply:
![image](https://user-images.githubusercontent.com/5930429/46570920-da42cb80-c974-11e8-820c-9059543713fa.png)

Reply:
Yeah I see C

Reply:
That's why I don't like TS actually (

Reply:
I can update whole interface `Telegraf` and change `C` to the `TelegrafContext`, if you want.

Reply:
What about `R extends ContextMessageUpdate`???

Reply:
> I can update whole interface Telegraf and change C to the TelegrafContext, if you want.

Ask @dotcypress about it, please.

Reply:
@dotcypress, I see three options:
* change `C` to `TelegrafContext`
* change `C` to `TContext`
* leave it as it is.

For me, one of the last two options is better, because it'll be more understandable that it is generic.
What do you think about this?

Reply:
@vladimir-fed I think second option is totally fine(`TContext`)

Reply:
Y do u sink so?